kafka-connect/src/test/java/io/tabular/iceberg/connect/channel/CoordinatorTest.java

```diff
@@ -103,18 +111,30 @@ public void testCommitNoFiles() {
     assertThat(producer.history()).hasSize(2);
     assertCommitComplete(1, commitId, ts);
 
-    verify(appendOp, times(0)).commit();
-    verify(deltaOp, times(0)).commit();
+    List<Snapshot> snapshots = ImmutableList.copyOf(table.snapshots());
+    Assertions.assertEquals(0, snapshots.size());
   }
 
   @Test
   public void testCommitError() {
-    doThrow(RuntimeException.class).when(appendOp).commit();
-
-    coordinatorTest(ImmutableList.of(EventTestUtil.createDataFile()), ImmutableList.of(), 0L);
+    // this spec isn't registered with the table
```

Comment:
Not clear to me here what exactly is being tested

Reply:
Old test was testing an exception, now we don't have that?

Reply:
So the old test was _effectively_ just making sure that if we get an error at `.commit()` time, do we leave the table still in a good state, i guess. 

The best I can do without mocks is give it bad files which causes `.appendFiles` to throw an error. 
So its a little different from the old test in that the error thrown now doesn't get thrown exactly at `.commit()` but i think the spirit is the same. 

Thoughts?

Reply:
I think that's good enough